docfx_project/docs/create-connector.md

```diff
@@ -0,0 +1,225 @@
+# Creating your first simulator connector
+
+#### Prerequisites
+  - .NET LTS version
+  - Cognite Data Fusion project 
+
+#### Create a new simulator connector project
+
+The Cognite Simulator Utils can be downloaded from [NuGet](https://www.nuget.org/packages/Cognite.Simulator.Utils/).
+
+To create a console application and add the latest version of the library, open the terminal and run the following commands:
+```sh
+dotnet new console -o NewSimConnector
+cd NewSimConnector
+dotnet add package Cognite.Simulator.Utils --prerelease
+```
+NB: The `--prerelease` flag is required to install the latest version of the library (alpha).
+
+#### Create a configuration file
+
+Create a `config.yml` file containing the simulator configuration
+
+```yaml
+version: 1
+
+logger:
+    console:
+        level: "debug"
+
+cognite:
+    project: ${COGNITE_PROJECT}
+    # This is for Microsoft Entra as an IdP, to use a different provider,
+    # set implementation: Basic, and use token-url instead of tenant.
+    # See the example config for the full list of options.
+    idp-authentication:
+        # Directory tenant
+        tenant: ${COGNITE_TENANT_ID}
+        # Application Id
+        client-id: ${COGNITE_CLIENT_ID}
+        # Client secret
+        secret: ${COGNITE_CLIENT_SECRET}
+        # List of resource scopes, ex:
+        # scopes:
+        #   - https://api.cognitedata.com/.default
+        scopes:
+          - ${COGNITE_SCOPE}
+
+
```

Comment:
suggestion: we could mention the `automation` section as well in the config + have a part of the tutorial where the connector uses something from the AutomationConfig, so users know they can define custom stuff in the Config.

Reply:
I think this would require another tutorial (as an another MD page), so maybe we could give more complex examples later?
Currently, this is rather simple tutorial, I agree.
I just can't really come up with any custom stuff idea for the Excel currently...